app/views/facilities/_product_list.html.haml

```diff
@@ -1,18 +1,22 @@
 - if products.any?
   .product_list{ class: products.first.class.model_name.to_s.pluralize.downcase }
-    %h3= product_list_title products, local_assigns[:title_extra]
+
+    - if current_facility.present?
+      %h3= product_list_title products, local_assigns[:title_extra]
+
     - if products.first.class.model_name == "TimedService"
       %p= t("facilities.show.timed_services_note")
     %ul
       - products.sort.each do |product|
         %li{ class: product.class.model_name.to_s.downcase }
-          = public_calendar_link(product)
+          - if current_facility.present?
```

Comment:
Does `public_calendar_link` require the facility? I think we can leave that in, I don't think having the calendar icon appear hurts anything, and it might be useful.

Reply:
I agree, if we can include it here, I say we do it.

Reply:
The `public_calendar_link` does require a facility, it links to `facility_instrument_public_schedule_path`.  There's a way to make it work though. I'll put it in my next commit with the other changes so you can let me know what you think.